src/uu/sum/src/sum.rs

```diff
@@ -126,7 +126,11 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
 
         let mut stdout = stdout().lock();
         if print_names {
-            writeln!(stdout, "{sum:0width$} {blocks:width$} {file}")?;
+            writeln!(
+                stdout,
+                "{sum:0width$} {blocks:width$} {}",
+                file.to_string_lossy()
```

Comment:
This is technically wrong, you should print the bytes (but you could just add a TODO).